sdk/cognitiveservices/azure-cognitiveservices-inkrecognizer/setup.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import re
+import os.path
+from io import open
+from setuptools import find_packages, setup  # type: ignore
+
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "azure-cognitiveservices-inkrecognizer"
+PACKAGE_PPRINT_NAME = "Ink Recognizer"
+
+# a-b-c => a/b/c
+package_folder_path = PACKAGE_NAME.replace('-', '/')
+# a-b-c => a.b.c
+namespace_name = PACKAGE_NAME.replace('-', '.')
+
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, '_version.py'), 'r') as fd:
+    version = re.search(r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]',  # type: ignore
+                        fd.read(), re.MULTILINE).group(1)
+
+if not version:
+    raise RuntimeError('Cannot find version information')
+
+with open('README.md', encoding='utf-8') as f:
+    readme = f.read()
+with open('HISTORY.md', encoding='utf-8') as f:
+    history = f.read()
+
+setup(
+    name=PACKAGE_NAME,
+    version=version,
+    description='Microsoft Azure {} Library for Python'.format(PACKAGE_PPRINT_NAME),
+    long_description=readme + '\n\n' + history,
+    long_description_content_type='text/markdown',
+    license='MIT License',
+    author='Microsoft Corporation',
+    author_email='azpysdkhelp@microsoft.com',
+    url='https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/cognitiveservices/azure-cognitiveservices-inkrecognizer',
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'License :: OSI Approved :: MIT License',
+    ],
+    zip_safe=False,
+    packages=find_packages(exclude=[
+        'tests',
+        # Exclude packages that will be covered by PEP420 or nspkg
+        'azure',
+    ]),
+    install_requires=[
+        'six>=1.6',
```

Comment:
@johanste I don't think any package are using six at this point of time?

Reply:
not sure about that
```
./core/azure-core/azure/core/pipeline/policies/distributed_tracing.py:
   29 : from six.moves import urllib
./core/azure-core/azure/core/settings.py:
   36 : import six
  167 :     if not isinstance(value, six.string_types):
./cosmos/azure-cosmos/azure/cosmos/_base.py:
   32 : import six
   33 : from six.moves.urllib.parse import quote as urllib_quote
  305 :     if not six.PY2:
  314 :         if not six.PY2:
  591 :         if six.PY2:
./cosmos/azure-cosmos/azure/cosmos/_consistent_hash_ring.py:
   26 : import six
   27 : from six.moves import xrange
  117 :         if isinstance(partition_key, six.string_types):
./cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py:
   28 : import six
  167 :             url = six.moves.urllib.parse.urlparse(host)
 2704 :             if not isinstance(query_body, dict) and not isinstance(query_body, six.string_types):
 2708 :             if isinstance(query_body, six.string_types):
 2712 :             and not isinstance(query_body, six.string_types)
./cosmos/azure-cosmos/azure/cosmos/_execution_context/document_producer.py:
   28 : import six
  156 :         if isinstance(val, six.string_types):
  179 :         if isinstance(val, six.string_types):
./cosmos/azure-cosmos/azure/cosmos/_execution_context/execution_dispatcher.py:
   26 : from six.moves import xrange
./cosmos/azure-cosmos/azure/cosmos/_execution_context/query_execution_info.py:
   25 : import six
   80 :         if isinstance(path, six.string_types):
./cosmos/azure-cosmos/azure/cosmos/_global_endpoint_manager.py:
   27 : from six.moves.urllib.parse import urlparse
./cosmos/azure-cosmos/azure/cosmos/_murmur_hash.py:
   26 : from six.moves import xrange
./cosmos/azure-cosmos/azure/cosmos/_partition.py:
   25 : from six.moves import xrange
./cosmos/azure-cosmos/azure/cosmos/_routing/collection_routing_map.py:
   27 : from six.moves import xrange
./cosmos/azure-cosmos/azure/cosmos/_synchronized_request.py:
   27 : from six.moves.urllib.parse import urlparse
   28 : import six
   59 :     if data is None or isinstance(data, six.string_types) or _is_readable_stream(data):
   65 :         if six.PY2:
  152 :     if not six.PY2:
  213 :     if request.data and isinstance(request.data, six.string_types):
./cosmos/azure-cosmos/azure/cosmos/auth.py:
   29 : import six
   96 :     if six.PY2:
  128 :             return next(six.itervalues(resource_tokens))
./cosmos/azure-cosmos/azure/cosmos/container.py:
   27 : import six
   95 :         if isinstance(item_or_link, six.string_types):
  101 :         if isinstance(conflict_or_link, six.string_types):
./cosmos/azure-cosmos/azure/cosmos/cosmos_client.py:
   27 : import six
   55 :     if isinstance(credential, six.string_types):
  193 :         if isinstance(database_or_id, six.string_types):
./cosmos/azure-cosmos/azure/cosmos/database.py:
   27 : import six
   82 :         if isinstance(container_or_id, six.string_types):
   96 :         if isinstance(user_or_id, six.string_types):
./cosmos/azure-cosmos/azure/cosmos/scripts.py:
   27 : import six
   57 :         if isinstance(script_or_id, six.string_types):
./cosmos/azure-cosmos/azure/cosmos/user.py:
   29 : import six
   52 :         if isinstance(permission_or_id, six.string_types):
./cosmos/azure-cosmos/samples/NonPartitionedCollectionOperations/Program.py:
   25 : import six
   28 : from six.moves.urllib.parse import quote as urllib_quote
   96 :         if not six.PY2:
  120 :         if not six.PY2:
./cosmos/azure-cosmos/setup.py:
   72 :       'six >=1.6',
./cosmos/azure-cosmos/test/aggregate_tests.py:
   28 : from six import with_metaclass
   29 : from six.moves import xrange
./cosmos/azure-cosmos/test/crud_tests.py:
   31 : from six.moves import xrange
   33 : # from six.moves.builtins import *
   36 : import six
   38 : if six.PY2:
./cosmos/azure-cosmos/test/globaldb_tests.py:
   22 : from six.moves.urllib.parse import urlparse
   23 : import six
./cosmos/azure-cosmos/test/location_cache_tests.py:
   15 : import six
   23 :         if six.PY2:
./cosmos/azure-cosmos/test/orderby_tests.py:
   31 : from six.moves import xrange
./cosmos/azure-cosmos/test/partition_key_tests.py:
   26 : import six
   29 : from six.moves.urllib.parse import quote as urllib_quote
   78 :         if not six.PY2:
  102 :         if not six.PY2:
./cosmos/azure-cosmos/test/proxy_tests.py:
   27 : import six
   28 : if six.PY2:
   48 :         if six.PY2:
./cosmos/azure-cosmos/test/query_execution_context_tests.py:
   25 : from six.moves import xrange
./eventhub/azure-eventhubs/azure/eventhub/common.py:
   11 : import six
  236 :             return six.text_type(data)
  389 :         elif isinstance(self.value, six.integer_types):
./eventhub/azure-eventhubs/azure/eventhub/error.py:
    6 : import six
   84 :         self.message = error_list if isinstance(error_list, six.text_type) else error_list.decode('UTF-8')
```

Reply:
remove six